src/client.ts

```diff
@@ -91,6 +92,21 @@ export class SignalSciencesAPIClient {
     }
   }
 
+  public async iterateCloudWAFInstances(
+    corpName: string,
+    iteratee: ResourceIteratee<SignalSciencesCloudWAF>,
+  ): Promise<void> {
+    const endpoint = this.buildEndpoint(corpName, '/cloudwafInstances');
+
+    const { data } = await this.fetch(endpoint, {
```

Comment:
This does not appear to currently support pagination. Did you see anything in the responses to indicate that pagination was supported/needed?

Reply:
Nothing about pagination, it just returns the data, similar to the other SS endpoints.
[The documentation](https://docs.fastly.com/signalsciences/api/#_corps__corpName__cloudwafInstances_get) also doesn't say anything about pagination.